glean-core/rlb/src/dispatcher/mod.rs

```diff
@@ -199,52 +199,57 @@ impl Dispatcher {
 
         let queue_preinit = Arc::new(AtomicBool::new(true));
 
-        let worker = thread::spawn(move || {
-            if block_receiver.recv().is_err() {
-                // The other side was disconnected.
-                // There's nothing the worker thread can do.
-                log::error!("The task producer was disconnected. Worker thread will exit.");
-                return;
-            }
-
-            let mut receiver = preinit_receiver;
-            loop {
-                use Command::*;
-
-                match receiver.recv() {
-                    Ok(Shutdown) => {
-                        break;
-                    }
-
-                    Ok(Task(f)) => {
-                        (f)();
-                    }
-
-                    Ok(Swap(swap_done)) => {
-                        // A swap should only occur exactly once.
-                        // This is upheld by `flush_init`, which errors out if the preinit buffer
-                        // was already flushed.
-
-                        // We swap the channels we listen on for new tasks.
-                        // The next iteration will continue with the unbounded queue.
-                        mem::swap(&mut receiver, &mut unbounded_receiver);
-
-                        // The swap command MUST be the last one received on the preinit buffer,
-                        // so by the time we run this we know all preinit tasks were processed.
-                        // We can notify the other side.
-                        swap_done
-                            .send(())
-                            .expect("The caller of `flush_init` has gone missing");
-                    }
+        let worker = thread::Builder::new()
+            .name("glean dispatcher".into())
```

Comment:
You use dots in the other names, so let's do it here too

```suggestion
            .name("glean.dispatcher".into())
```